certbot-apache/certbot_apache/obj.py

```diff
@@ -18,6 +19,9 @@ def __eq__(self, other):
                      self.is_wildcard() and other.is_wildcard()))
         return False
 
+    def __repr__(self):
+        return "certbot_apache.obj.Addr(" + repr(self.tup) + ")"
+
     def __ne__(self, other):
         return not self.__eq__(other)
```

Comment:
I'm mildly distressed by `__repr__` inserting itself between `__eq__` and `__ne__`, since those two maybe ought to go together.